react-challenge.html

```diff
@@ -32,27 +32,31 @@
             currentMood: ""
           }
         }
-
+        
         handleMoodOptions = () => {
+          let { moods } = this.state
           let randomIndex = Math.floor(Math.random() * this.state.moods.length )
```

Comment:
```suggestion
          let randomIndex = Math.floor(Math.random() * moods.length )
```
Destructured variable can me used here too :)